src/media/css/elements--header.styl

```diff
@@ -145,7 +211,7 @@ mkt-header-child-toggle {
 }
 
 @media $base-desktop {
```

Comment:
Would be nice to move this into the `.mkt-header-child--link-list` block with `+media--base-desktop()`.